lib/natural/classifiers/classifier.js

```diff
@@ -34,6 +34,15 @@ var Classifier = function(classifier, stemmer) {
 };
 
 function addDocument(text, classification) {
+    
+    // Ignore further processing if classification is undefined 
+    if(typeof classification === 'undefined') return;
+    
+    // If classification is type of string then make sure it's dosen't have blank space at both end  
+    if(typeof classification === 'sytring'){
```

Comment:
It is ok.  I'm not a contributor, just like to lurk around possible projects that I like, perhaps might use for a future project of mine.